ui/src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
```

Comment:
I have something similar coming in my branch => https://github.com/aerogear/mobile-client-service/blob/over-list-base/ui/src/components/common/ComponentSectionLabel.js / https://github.com/aerogear/mobile-client-service/blob/over-list-base/ui/src/components/common/ComponentSectionLabel.scss

(Naming could probbaly be improved) but it is what is defined in openshift currently

Reply:
Ok, should I leave it like this for now and update it so that it uses your component later?

Reply:
:+1: